### PR TITLE
:bug: add missing python-distutils dependency

### PR DESCRIPTION
**Pull request by ybelMekk**
Add python3-utils, making the build of crash.

### CODE REVIEW

**Comment by bgauduch** [comment]
Hello @ybelMekk, thanks for opening this PR !

I'm not sure what you mean, the image build is ok without the python3-utils dependency as you can see [here](https://github.com/Zenika/terraform-azure-cli/runs/673802929?check_suite_focus=true)

What issue do you have ?

**Comment by bgauduch** [comment]
Ok I've just seen the issue #73

File: Dockerfile [hunk 1]
```diff
@@ -27,6 +27,7 @@ FROM debian:buster-20191118-slim as azure-cli
 ARG AZURE_CLI_VERSION
 ARG PYTHON_MAJOR_VERSION
 RUN apt-get update
+RUN apt-get install -y python3-distutils
```
Comment (bgauduch) [inline Dockerfile:30]: @ybelMekk `python3-distutils` do the trick but it need to be moved out of this build step to the final image like so :

```docker
RUN apt-get update \
  && apt-get install -y --no-install-recommends \
    ca-certificates=20190110 \
    git=1:2.20.1-2+deb10u3 \
    python3=${PYTHON_MAJOR_VERSION}.3-1 \
   python3-distutils=3.7.3-1 \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* \
  && update-alternatives --install /usr/bin/python python /usr/bin/python${PYTHON_MAJOR_VERSION} 1
```

Comment (ybelMekk) [reply Dockerfile:30]: Okei, fixing it

**Comment by ybelMekk** [comment]
resolved the changes

**Comment by bgauduch** [comment]
> resolved the changes

Thanks a lot for the fix @ybelMekk, tested and approved !

**Pull request merged**